cypress/test/newsroom/NewsroomSection.cy.js

```diff
@@ -0,0 +1,25 @@
+import { mount } from 'cypress/react'
+import NewsroomSection from '../../../components/newsroom/NewsroomSection';
+const newsroom = ["Latest news and blogs", "Welcome to our Newsroom section. Here, you'll get latest information about our blogs, articles, announcements and Youtube live-streams. Let's get upto date with the recent activities in the organization."]
+describe('Features Component', () => {
+    it('renders without errors', () => {
+        mount(<NewsroomSection />);
```

Comment:
Use `beforeEach` function of cypress to avoid repition of mount function.

Applicable everywhere.

Reply:
done